compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIELinalgFunctionOutlining.cpp

```diff
@@ -91,6 +96,7 @@ class AMDAIELinalgFunctionOutliningPass
 };
 
 void AMDAIELinalgFunctionOutliningPass::runOnOperation() {
+  if (!enableFunctionOutlining) return;
```

Comment:
There's no point to use this flag within the pass. This should only be controlled outside in passes.cpp.

Reply:
Make sense I think, like the `enableVectorizationPasses` approach.

Reply:
Really nice point! Thanks @yzhang93 !